Package.swift

```diff
@@ -16,6 +16,10 @@ let package = Package(
     dependencies: [
         .package(url: "https://github.com/apple/swift-collections.git", from: "1.1.0"),
         .package(url: "https://github.com/apple/swift-log.git", from: "1.5.4"),
+        // NOTE: Compiling to wasm requires specific versions of swift-nio.
```

Comment:
I'm happy to bump this to a version of NIO that supports Wasm. There's no issue bumping non-major versions of dependencies

Reply:
@0xTim My [fix for NIOCore](https://github.com/apple/swift-nio/pull/3271) is now in a NIO release, `2.84.0`. I think the most appropriate thing to do here per your comment is to bump the nio minor version.

> There's no issue bumping non-major versions of dependencies

If the [NIO wasm CI PR](https://github.com/apple/swift-nio/pull/3159) ever merges, we can make another pass to make that the new minimum.

Reply:
I bumped the minor NIO version.